### ISSUE TITLE
Milestone 1: Proof of concept for developer and user experience

### ISSUE THREAD

**Issue by steffen** [rarely-org/rarely#3]
This milestone is all about getting started with the core functionality. UI aspects such as design, usability, accessibility and style should be “ignored” for this milestone.
This will change in future milestones where design, usability, accessibility, style will be very important.

**Goal:**
Proof of concept for developer and user experience with proposed technology stack, see proposal for [frontend](https://github.com/rarely-org/rarely/discussions/4) and [backend](https://github.com/rarely-org/rarely/discussions/6).

**Features:**
- [Authentication](https://github.com/rarely-org/rarely/issues/7)
- [Create and list items](https://github.com/rarely-org/rarely/issues/8)

---

Each issue for this milestone will be tracked in a project.
This issue will serve for high-level discussion and status updates for this milestone.

**Comment by steffen** [comment]
Closing this issue in favor of the project: https://github.com/orgs/rarely-org/projects/1/views/1

Status updates may be shared via releases. Let's use issues for actionable items only.
